### PR TITLE
Expanded restriction examples

### PR DESCRIPTION
**Pull request by allisonchihak**
Added examples of each supported restriction in Mews to make the scenarios we support more clear.

### CODE REVIEW

**Comment by patrikmiksovsky95** [comment]
@allisonchihak is this still relevant with ksenia PR ?

File: channel-manager-api.md [hunk 1]
```diff
@@ -139,7 +139,119 @@ Mews automatically sends changes in Inventory \(once connection is set up\). Dat
...
-#### Request `[ChannelManagerApiAddress]/updateRestrictions`
+#### Request Examples `[ChannelManagerApiAddress]/updateRestrictions`
+
+**Stop Sell Restrictions**
+
+- Stop Sell with no exceptions
+
+```javascript
+{
+   "clientToken":"[Mews Client token]",
```
Comment (vitasamek) [inline channel-manager-api.md]: is this format of the json which is used everywhere? it is incorrectly formatted. (spaces missing e.g)

File: channel-manager-api.md [hunk 2]
```diff
@@ -139,7 +139,119 @@ Mews automatically sends changes in Inventory \(once connection is set up\). Dat
...
+         "state":[
+            2,
+            8
+         ]
+      }
+   ]
+}
+```
+
+- Stop Sell for reservations of 3 nights or less
```
Comment (vitasamek) [inline channel-manager-api.md]: missing a dot.

File: channel-manager-api.md [hunk 3]
```diff
@@ -139,7 +139,119 @@ Mews automatically sends changes in Inventory \(once connection is set up\). Dat
...
+         ],
+         "maxLos":14
+      }
+   ]
+}
+```
+
+**Closed to Arrival Restrictions**
+
+-Closed to Arrival with no exceptions
```
Comment (vitasamek) [inline channel-manager-api.md]: a dot

File: channel-manager-api.md [hunk 4]
```diff
@@ -139,7 +139,119 @@ Mews automatically sends changes in Inventory \(once connection is set up\). Dat
...
+            2,
+            8
+         ],
+         "maxLos":14
+      }
+   ]
+}
+```
+
+**Closed to Arrival Restrictions**
```
Comment (vitasamek) [inline channel-manager-api.md]: IMHO incorrect, there should be a title

File: channel-manager-api.md [hunk 5]
```diff
@@ -139,7 +139,119 @@ Mews automatically sends changes in Inventory \(once connection is set up\). Dat
...
+         "state":[
+            2,
+            6
+         ]
+      }
+   ]
+}
+```
+
+- Closed to Arrival for reservations of 2 nights or less
```
Comment (vitasamek) [inline channel-manager-api.md]: is this description? Where is a title?

File: channel-manager-api.md [hunk 6]
```diff
@@ -149,35 +261,127 @@ Mews automatically sends changes in Inventory \(once connection is set up\). Dat
...
+            2,
+            6
+         ],
+         maxLos: 7
+      }
+   ]
+}
+```
+
+**Closed to Departure Restrictions
```
Comment (vitasamek) [inline channel-manager-api.md]: IMHO title again.

File: channel-manager-api.md [hunk 7]
```diff
@@ -149,35 +261,127 @@ Mews automatically sends changes in Inventory \(once connection is set up\). Dat
...
+
+- Closed to Departure with no exceptions
```
Comment (vitasamek) [inline channel-manager-api.md]: a dot is missing again

File: channel-manager-api.md (hunk 6 above)
Comment (vitasamek) [reply channel-manager-api.md]: and the same issues bellow.

**Pull request closed without merging**